### PR TITLE
Enable CI on Appveyor

### PR DESCRIPTION
**Pull request by kenhys**

### CODE REVIEW

**Comment by kenhys** [comment]
@abetomo could you review this PR, please?

File: lib/nroonga.js [hunk 1]
```diff
@@ -1,2 +1,6 @@
 'use strict'
+if (process.env.GROONGA_PATH) {
+  const path = require('path')
+  process.env.PATH = process.env.GROONGA_PATH + path.sep + 'bin' + path.delimiter + process.env.PATH
```
Comment (abetomo) [inline lib/nroonga.js]: If you use `path`, it is better to use `path.join()`.

**Comment by abetomo** [comment]
Thank you for PR!

File: lib/nroonga.js (hunk 1 above)
Comment (kenhys) [reply lib/nroonga.js]: @abetomo Thanks, I've fixed it!

**Pull request merged**